nixos/modules/services/hardware/mdcheck.nix

```diff
@@ -0,0 +1,62 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.mdcheck;
+in
+{
+  options.services.mdcheck = {
+    enable = mkEnableOption (lib.mdDoc ''
+      This option enables mdcheck timers that run regular full scrubs of the
+      md devices.  These processes can also cause high I/O utilization; the
+      configuration option `boot.kernel.sysctl."dev.raid.speed_limit_max"`
+      can be used to limit the I/O utilization of the mdcheck processes.
```

Comment:
We should do a nix option for that

Reply:
@SuperSandro2000 Why? It would just be trading one oneliner for another.

Reply:
To make it discoverable. If we already mention a particular setting in a description than this is a nice addition.